package.json

```diff
@@ -16,6 +17,7 @@
     "@types/react-redux": "^7.1.7",
     "@types/react-router-dom": "^5.1.3",
     "awesome-typescript-loader": "^5.2.1",
+    "axios": "^0.19.2",
```

Comment:
Is this a peer dependency for mapbox? I don't see it used anywhere else.
If it's not we should remove it.

We should be careful not to add packages for single use things as it can quickly bloat the app and make the site slower due to larger bundle sizes